### PR TITLE
Rename quicperf driver to quicperfdrv, add private instance

### PR DESCRIPTION
**Pull request by ThadHouse**
Changing the name makes symbols not interfere between executable and driver, and we need a version that links into shipping msquic for internel tests

### CODE REVIEW

File: scripts/performance-helper.psm1 [hunk 1]
```diff
@@ -342,7 +342,7 @@ function Invoke-RemoteExe {
     } -ArgumentList $Exe
 
     if ($Kernel) {
-        $RunArgs = "--kernel $RunArgs"
+        $RunArgs = "--kernel --privateLibrary $RunArgs"
```
Comment (nibanks) [inline scripts/performance-helper.psm1:345]: Another option is have `--kernel` and `--kernelPriv` options.

File: src/perf/bin/appmain.cpp [hunk 2]
```diff
@@ -23,7 +23,8 @@
 #include "PerfIoctls.h"
 #include "quic_driver_helpers.h"
 
-#define QUIC_DRIVER_NAME   "quicperf"
+#define QUIC_DRIVER_NAME   "quicperfdrv"
+#define QUIC_DRIVER_NAME_PRIVATE    "quicperfdrvpriv"
```
Comment (nibanks) [inline src/perf/bin/appmain.cpp:27]:
```suggestion
#define QUIC_DRIVER_NAME                   "quicperfdrv"
#define QUIC_DRIVER_NAME_PRIVATE    "quicperfdrvpriv"
```

**Review by nibanks — APPROVED** [review]

File: scripts/performance-helper.psm1 (hunk 1 above)
Comment (ThadHouse) [reply scripts/performance-helper.psm1:345]: That likely is better. I would likely change the tests at the same time as well so they match. This was pulled straight from the tests.

Comment (nibanks) [reply scripts/performance-helper.psm1:345]: I leave it up to you to do now or later.

File: src/perf/bin/appmain.cpp [hunk 3]
```diff
@@ -371,6 +371,9 @@ main(
         if (strcmp("--kernel", argv[i]) == 0) {
```
Comment (nibanks) [inline src/perf/bin/appmain.cpp:371]: Shouldn't you be checking for `--kernelPriv` too here?

File: src/test/bin/quic_gtest.cpp [hunk 4]
```diff
@@ -985,9 +985,9 @@ int main(int argc, char** argv) {
     for (int i = 0; i < argc; ++i) {
         if (strcmp("--kernel", argv[i]) == 0) {
```
Comment (nibanks) [inline src/test/bin/quic_gtest.cpp:986]: Shouldn't you be checking for `--kernelPriv` here too?

Comment (ThadHouse) [reply src/test/bin/quic_gtest.cpp:986]: No need. `--kernel` strcmp will match `--kernelPriv`

Comment (nibanks) [reply src/test/bin/quic_gtest.cpp:986]: strcmp compares to the null terminator. They won't be equal.

**Pull request merged**